stages/org.osbuild.machine-id

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+"""
+Deal with /etc/machine-id
+
+Explicitly define the state to /etc/machine-id.
+"""
```

Comment:
This docstring could be a bit more descriptive especially since there's so much nuance in what happens when certain strings are written.

Reply:
Now there's just one typo left ;)

Reply:
Thanks! I fixed one typo, I hope it was the one you are referring to :)